build.gradle

```diff
@@ -78,9 +77,29 @@ dependencies
     shaded project.configurations.getByName('compile')
     shaded packages.slf4j.log4j12
     shaded packages.log4j
+
+    // Support Junit 5 tests
+    testImplementation packages.junit.api
+    testRuntimeOnly packages.junit.engine
+    // Support JUnit 3/4 tests
+    compile packages.junit.junit4
+    testRuntimeOnly packages.junit.vintage
```

Comment:
I'll check that -- I was replacing the `compile packages.junit` earlier in the stanza. I *think* that needs to be kept (as `compile packages.junit.junit4` since some of the non-test classes uses JUnit4 in some of the downstream projects, and I presume that some non-test classes do the same in `atlas`. I'll check.).